src/Raven.Server/Documents/Handlers/BatchRequestParser.cs

```diff
@@ -329,7 +330,7 @@ private static void ThrowMissingPatchProperty()
             BlittableJsonReaderObject reader;
             using (var builder = new BlittableJsonDocumentBuilder(ctx,
                 BlittableJsonDocumentBuilder.UsageMode.ToDisk,
-                key, parser, state))
+                key, parser, state, modifier: new StreamSource.BlittableMetadataModifier(ctx)))
```

Comment:
To a shared location, since it isn't just smuggler only code, so shouldn't live in the smuggler class